src/Compilers/CSharp/Test/Semantic/Semantics/OverloadResolutionPerfTests.cs

```diff
@@ -0,0 +1,134 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using System.Linq;
+using System.Text;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
+{
+    public class OverloadResolutionPerfTests : CSharpTestBase
```

Comment:
>OverloadResolutionPerfTests [](start = 17, length = 27)

Is the idea that the tests would fail due to a timeout if there is a problem? Consider adding a comment about that.